File: web3rpc/rpc-specs/code-samples/javascript/klay/configuration/getChainConfig.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const blockNumberOrTag = 100
    provider.klay.getChainConfig({ blockNumberOrTag }, (err, data, response) => {
        console.log(data);
    });
}
)()

File: web3rpc/rpc-specs/code-samples/python/klay/configuration/getChainConfig.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

blockTag = 100

w3 = Web3(Web3.HTTPProvider(host))
klay_response = w3.klay.get_chain_config()

print(klay_response)
